lib/install/config/webpacker.yml

```diff
@@ -4,7 +4,7 @@ default: &default
   source_path: app/javascript
   source_entry_path: packs
   public_output_path: packs
-  compile_missing_packs: true
+  compile_missing_packs: false
```

Comment:
@javan Sure, that would make more sense or may be `test_compiler_enabled:` for clarity- I was looking at this for 5 minutes and finally had to refer to PR to understand what this does 👍